mantidproject/mantid
# Add options to ColorBarWidget to change shown checkboxes and their label/tooltip text

## Description of work.

The ColorBarWidget.cpp now has methods for changing both of the [checkboxes' label, tooltip text](https://github.com/mantidproject/mantid/commit/713a181a3bc51eb0b79dac859b0849e15402c8a3#diff-4aa05715847f2ffc407c1b4bd6c6ca59R381) and [visibility](https://github.com/mantidproject/mantid/commit/480f71ba1d4c22dbbb2931fef1201ea2e96ca792#diff-4aa05715847f2ffc407c1b4bd6c6ca59R108).
## To test:
- Go to `Interfaces` > `Diffraction` > `Tomographic Reconstruction`, go to tab `ROI etc.`
- Check that only **1** checkbox is shown, should be `Autoscale on load`, and that the tooltip is correct `"This flag signals that the color scale range will be set automatically to the current slice range when an image is loaded. Note that auto scaling will be applied when an image is loaded for the very first time."`
- Look for any spelling mistakes in the tooltip, I might've missed them
- Check that the Color Bar Widget behaves normally in the other places it's used (such as the Slice Viewer on a Workspace), and any other usages that I am not aware of.



Fixes #16154 .



_Does not need to be in the release notes._

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
